doalerts/perfherder_alerts.py

```diff
@@ -198,31 +117,138 @@ def filterUniqueAlerts(results):
     return results
 
 
-def analyzeTest(framework, branch, platform, testname, subtests):
-    signatures = getSignatures(branch, framework, platform)
-    sig_ids = filterSignatureIds(signatures, testname, subtests)
-
-    results = []
-    for sig in sig_ids:
-        result = analyzeData(sig, branch, framework)
-        if result:
-            results.append({"sig": sig, "result": result})
+class Alerts(object):
+    def __init__(
+        self, framework, branch, platforms, subtests, days, test, metrics, verbose
+    ):
+        self.framework = getFrameworkId(framework)
+        self.branch = branch
+        self.platforms = platforms
+        self.subtests = subtests
+        self.interval = 86400 * days
+        self.test = re.compile(test)
+        self.metrics = metrics
```

Comment:
geomean, fcp, loadtime, etc. It's the name of the value, and allows me to run for example:

```
python perfherder_alerts.py -f raptor -b mozilla-inbound -p linux64 -t tp6 --subtests -m geomean -m fcp
```

This will skip analysing all the other metrics, and only show geomean and fcp.